man/paru.conf.5

```diff
@@ -1,55 +1,55 @@
 '\ t
-.TH "PARU.CONF" "5" "2019\-10\-21" "paru v0.0.1" "Paru  Manual"
+.TH "PARU.CONF" "5" "2020\-10\-28" "paru v1.0.0" "Paru Manual"
 .nh
 .ad l
 .SH NAME
-PARU \- AUR Helper
+paru.conf \- paru configuration file
 
 .SH SYNOPSIS
-$PARU_CONF
-.sp
-/etc/paru.conf
-.sp
-~/.config/paru/paru.conf
+/etc/paru.conf, $XDG_CONFIG_HOME/paru/paru.conf, $HOME/.config/paru/paru.conf
```

Comment:
You left you `$PARU_CONF` when you merged changes.

Reply:
When $PARU_CONF is set paru reads the file from $PARU_CONF/paru.conf, right?

Reply:
Nope, `$PARU_CONF` is the `paru.conf` file.

Reply:
Ok. I'm going to update the description with that then.

Reply:
When the file at $PARU_CONF doesn't exist, paru exits with an somewhat useless error `error: No such file or directory (os error 2)`. Should I mention this in the man page, or should a better error message be added?

Reply:
Bad error message is an issue with the program. It should be fixed there.